cli.js

```diff
@@ -4,17 +4,24 @@ const { hideBin } = require("yargs/helpers");
 const fs = require("node:fs/promises");
 const path = require("node:path");
 
+const { GITHUB_API_URL } = require("./util/constants");
+
 (async () => {
     const scriptsDir = path.join(__dirname, "scripts");
     const files = await fs.readdir(scriptsDir);
 
     let argv = yargs(hideBin(process.argv))
-        .option("t", {
-            alias: "token",
+        .option("token", {
+            alias: "t",
             describe: "GitHub personal access token to use when making API calls",
             demandOption: true,
         })
-        .default("t", () => process.env.GITHUB_TOKEN, "GITHUB_TOKEN environment variable");
+        .option("api-url", {
+            alias: "u",
+            describe: "When using with GitHub Enterprise Server, set this to the root URL of the API. Example: https://github.my-domain.com/api/v3",
+        })
+        .default("token", () => process.env.GITHUB_TOKEN, "GITHUB_TOKEN environment variable")
+        .default("api-url", () => process.env.GITHUB_API_URL || GITHUB_API_URL, `GITHUB_API_URL environment variable, or ${GITHUB_API_URL} if not specified`);
```

Comment:
defaults to the environment variable `GITHUB_API_URL`, or the constant `https://api.github.com` if that isn't specified.